libs/designsystem/src/lib/components/button/button.component.scss

```diff
@@ -101,7 +101,7 @@
   // Only apply focus ring if pointer device can hover
   // (effectively desktop/mouse devices):
   @include focus() {
-    border-color: $focus-ring-color;
+    border-color: $focus-ring-color !important;
```

Comment:
In general it's not the preferred way to use `!important` declarations to override the cascade. We only use this when overriding 3rd party libs where we are unable to alter the css specificity (by specific selectors and/or add css classes to the element in question).
This is also the reason for the original selector (`&:not(:focus):not(:disabled)`)
Couldn't this be done another way? 🤔